src/mumble/UserLocalVolumeSlider.cpp

```diff
@@ -0,0 +1,50 @@
+// Copyright 2021 The Mumble Developers. All rights reserved.
```

Comment:
All copyright headers need to be updated sometime before merge. Just use `2022` as the creation year.